Readme.md

```diff
@@ -23,7 +23,7 @@ file, or invoke the following commands in developer console:
 ```console
 $ nuget restore
 $ msbuild
-$ powershell scripts/deps-windows.ps1 # consult the script documentation, there're additional parameters
+$ fsi scripts/deps-downloader.fsx win {Build_Type}(debug/release) # consult the script documentation, there're additional parameters
```

Comment:
1. The script should setup both Debug and Release configurations. Please remove the second argument here.
2. Remove the comment about the documentation; we have no other parameters or documentation.